CHANGELOG

```diff
@@ -3,10 +3,25 @@ Changelog
 
 This document describes changes between each past release.
 
-0.15.0 (unreleased)
+0.15.0 (2015-01-08)
 -------------------
 
-- No changes yet.
+- [feature] Handle the channel parameter for rooms (1094852)
+- [tools] Update the script to estimate the redis usage wrt user count (1067401)
+- [tools] Add a script to remove broken session fixed by 0.14.2 (1111579)
+- [feature] Add PATCH /rooms that let you delete multiple rooms at once (1106562)
+- [config] Change logging from winston to mozlog - logs are now with heka format on STDOUT (1080133)
+- [feature] Send a message to sentry on server startup to make sure sentry is configured (1098241)
+- [metrics] Logs number of participants in rooms metrics (1112379)
+- [metrics] Do not broadcast terminate:close if another terminate message has been logged (1103898)
+- [bugfix] Update loadtest with the new room's owner guarented seat. (1111655)
+- [feature] Improve trades between redis and loop-server using multi. (1100903)
+- [migration] Remove old 0.13 migration code. (1069208)
+- [tools] Rewrite all redis tools in Javascript to benefit from the server config. (1079812)
+- [feature] Let the future participant grab information about the room before entering it. (1103331)
+- [feature] Show deleted rooms only if a version has been specified. (1106902)
+- [feature] Limit length of room subjects to 124 characters. (1107082)
+- [tools] Add a smoketest target for the loadtest.
```

Comment:
Actually, doing this in a pull request is something we should do more often. Not only this helps everyone to get a grasp of what landed since the last tag, but we also can review the format of the changelog!

In this case, I feel that we should be regrouping the entries by category, in order to ease the reading (e.g. get all of the features together, etc.)